libraries/AP_Scripting/applets/Aerobatics/FixedWing/plane_aerobatics.lua

```diff
@@ -117,7 +161,7 @@ if TRIK_ENABLE:get() > 0 then
    for i = 1, count do
       local k = 5*i
       local prefix = string.format("%u", i)
-      TRICKS[i] = TrickDef(bind_add_param2(prefix .. "_ID",   k+0, i),
+      TRICKS[i] = TrickDef(bind_add_param2(prefix .. "_ID",   k+0, -1),
                            bind_add_param2(prefix .. "_ARG1", k+1, 30),
```

Comment:
I think maybe arg1 should not default to a valid/non-zero value. Someone might leave it at the default potentially causing unexpected behavior.

Reply:
its the only way to get the "not set up"....leaving at default  -1 means the trick does nothing